FieldOpt/Settings/model.cpp

```diff
@@ -55,9 +56,32 @@ Model::Model(QJsonObject json_model, Paths &paths)
     if (!json_model.contains("ControlTimes") || !json_model["ControlTimes"].isArray())
         throw UnableToParseModelSectionException("The ControlTimes array must be defined with at least one time for the model.");
     control_times_ = QList<int>();
+    if (json_model.contains("NPVInterval")) {
+        if(json_model["NPVInterval"].toString().compare("Yearly")==0) {
+            if (json_model.contains("NPVYears")) {
+                for (int i = 0; i < json_model["NPVYears"].toInt(); ++i) {
+                    control_times_.append(365 * i);
+                }
+            } else {
+                throw UnableToParseModelSectionException("Unable to parse NPVYears");
+            }
+
+        } else if (json_model["NPVInterval"].toString().compare("Monthly")==0) {
```

Comment:
The indentation in this new code is slightly off. Run auto indent on the file (after changing the clion code style settings mentioned above)